_posts/2017/2017-11-12-use-a-pull-request-template.markdown

```diff
@@ -0,0 +1,67 @@
+---
+title: 'Use a pull request template'
+description: 'It helps you answer what, why & how.'
```

Comment:
Please use 'and' instead of '&'.

Maybe a 'nitpick' comment...I'm especially sensitive to these today as was working on an XML builder/RSS thing for Saron and having these in text caused ag as XML can't parse them!

Reply:
I want this to be as short a possible as it ends up in open graph tags. Plus I like ampersands. ;-)

Reply:
Fine! :P